### ISSUE TITLE
Development Setup Steps needs to be updated

### ISSUE THREAD

**Issue by bwittgen** [Sonarr/Sonarr#2988]
I am trying to setup a developer environment for Sonarr, however this "_info_" link is broken on this step on GitHub:

Clone the repository into your development machine. _info_

Sorry if this isn't the best way to report this. If someone could point me to this step and correct this I would appreciate it, thanks!

**Comment by markus101** [comment]
Since you didn't specify, I assume you're looking at the readme?

I'm not sure where that link is supposed to go because github pulled their own support article on the topic, maybe https://help.github.com/en/articles/working-with-forks but that's not quite the same.

**Comment by bwittgen** [comment]
Sorry no I mean on the main GItHub page:

Configuring Development Environment:
Requirements
Visual Studio 2015 (https://www.visualstudio.com/vs/)
Git
NodeJS
Setup
Make sure all the required software mentioned above are installed.
Clone the repository into your development machine. info
Grab the submodules git submodule init && git submodule update
Install the required Node Packages npm install
Start gulp to monitor your dev environment for any changes that need post processing using npm start command.
Please note gulp must be running at all times while you are working with Sonarr client source files.

**Comment by bwittgen** [comment]
But it looks like it's both of them ha!

**Comment by markus101** [comment]
Right, that comes from [README.md](https://github.com/Sonarr/Sonarr/blob/develop/README.md)

**Comment by markus101** [comment]
@Taloth should we switch this to https://help.github.com/en/articles/working-with-forks or just remove it? I realize not everyone is familiar with it, but if links are just going to die I don't think we want to chase them, we've got better things to do.

**Comment by bwittgen** [comment]
Honestly if the steps could be looked over and see if they could be a little more detailed that would be helpful. I am trying to get involved in this project but I am trying to familiarize myself with just getting the dev environment setup at this point.

**Comment by markus101** [comment]
Anything in particular?

We could probably expand on front end vs backend, but it's not meant as an exhaustive guide on setting up your environment for C#/JS development.

**Comment by Taloth** [comment]
@markus101 Sure, link to that new page.

**Comment by bwittgen** [comment]
1. Well first of all this step that I opened the bug for, but I should have this done with visual studio.
"Clone the repository into your development machine. info"

2. Grab the submodules git submodule init && git submodule update <-- Where am I supposed to run this? From what context?

3. Install the required Node Packages npm install <-- Again, where am I supposed to run this? From what context?

4. Start gulp to monitor your dev environment for any changes that need post processing using npm start command. <-- I assume this is running from the same context as above?

**Comment by markus101** [comment]
1. There are plenty of ways to clone a repository, Visual Studio is just one of them, each developer will have their own preference
2. Not sure if you mean where is terms of which app or where on your computer.


- Which app would be `cmd.exe`, git bash or anything else you use to run command line commands
- Where on your computer depends where you cloned the repository to. It's implied with all the projects I've looked at that it'll be the directory where you cloned to, I don't think this needs to be explicitly written out for Sonarr.

I've updated the readme for v3 in the `phantom-develop` branch, which is what we'll be using for any development work until v3 is released https://github.com/Sonarr/Sonarr/tree/phantom-develop
